packages/config/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+      "rootDir": "./src",
+      "outDir": "./",
+      "module": "commonjs",
+      "target": "ES5",
+      "declarationDir": "./",
+      "allowJs": true,
+      "resolveJsonModule": true,
+      "isolatedModules": true,
+      "plugins": [],
+      "lib": ["ES2015"]
+  },
+  "include": [
+      "./src/**/*"
+  ],
+  "exclude": [
+    "./src/**/*.test.js",
+    "./src/**/*.spec.js",
```

Comment:
Doesn't seem to be an issue now, but we also may want to catch the `.jsx` extension in the future